src/instruments/src/EFB/Ground/Pages/PushbackPage.tsx

```diff
@@ -616,6 +619,17 @@ export const PushbackPage = () => {
                             </div>
                         </TooltipWrapper>
                     </div>
+
+                    <div className={`flex flex-row items-center h-10 ${!pushbackActive && 'opacity-30'
+                    + ' pointer-events-none'}`}
```

Comment:
```suggestion
                    <div className={`flex flex-row items-center h-10 ${!pushbackActive && 'opacity-30 pointer-events-none'}`}
```

Reply:
Thanks - IDE does this things sometime and I missed this one